apache/Dockerfile

```diff
@@ -88,6 +88,15 @@ RUN set -ex; \
 	mv /usr/src/GeoIPCity/GeoLite2-City.mmdb /usr/src/piwik/misc/GeoLite2-City.mmdb; \
 	rm -rf GeoIPCity*
 
+RUN set -ex; \
+	curl -fsSL -o EnvironmentVariables.tar.gz \
+		"https://github.com/matomo-org/plugin-EnvironmentVariables/archive/3.0.0.tar.gz"; \
```

Comment:
use https://plugins.matomo.org/api/2.0/plugins/EnvironmentVariables/download/latest

BTW could you refactor this code so it's written only once instead of copy pasted multiple times?